src/components/CommentInputBox.jsx

```diff
@@ -0,0 +1,59 @@
+import { postComment } from '../api';
+import { useState, useContext } from 'react';
+import { UserContext } from '../contexts/LoggedInUser';
+
+export default function CommentInputBox({ refreshComments, article_id }) {
+
+    const [ commentBody, setComment ] = useState('');
+    const { submitComment, isLoading, error } = sendComment(refreshComments, article_id);
+    const { user } = useContext(UserContext);
+
```

Comment:
Currently the user can mash the submit button and submit loads of messages. I'd recommend adding a few features to help with this 
- Add some input validation so that a blank comment can't be submitted
- Make it so the submit button is disabled while a comment is in the process of being posted